File: app/src/release/java/com/kaciula/archiman/util/injection/OkHttpModule.java
package com.kaciula.archiman.util.injection;

import dagger.Module;
import dagger.Provides;
import javax.inject.Singleton;
import okhttp3.OkHttpClient;

@Module
public class OkHttpModule {

  @Provides
  @Singleton
  OkHttpClient provideOkHttpClient(OkHttpClient.Builder builder) {
    return builder.build();
  }
}

File: app/src/release/java/com/kaciula/archiman/injection/OkHttpModule.kt
package com.kaciula.archiman.injection

import dagger.Module
import dagger.Provides
import okhttp3.OkHttpClient
import javax.inject.Singleton

@Module
class OkHttpModule {

    @Provides
    @Singleton
    fun provideOkHttpClient(builder: OkHttpClient.Builder): OkHttpClient {
        return builder.build()
    }
}
